Runtime/Behaviours/CinemachineTargetGroup.cs

```diff
@@ -256,9 +256,6 @@ private static BoundingSphere WeightedMemberBounds(Target t, Vector3 avgPos, flo
         /// </summary>
         public void DoUpdate()
         {
-            if (IsEmpty)
```

Comment:
IsEmpty check removed, because we need to update mAveragePos and BoundingBox. But, when the list is empty, we only need to update once, and not more.
Should I modify the code to update only once?